.git-blame-ignore-revs

```diff
@@ -0,0 +1 @@
+8712af85d0c356565117461c0be392f1c618fe7a
```

Comment:
https://github.com/borgbackup/borg/blame/cf12e47aa0b1c282b0d6018cd8bfb3e130a9df35/conftest.py

seems like that is not used (yet?) by github blame.

Reply:
> seems like that is not used (yet?) by github blame.

Indeed
> https://black.readthedocs.io/en/stable/guides/introducing_black_to_your_project.html#avoiding-ruining-git-blame
> The one caveat is that GitHub and GitLab do not yet support ignoring revisions using their native UI of blame.

Reply:
maybe it needs to be merged to get active? we'll see.